### DISCUSSION TITLE
Has development / support of Link Map ceased?

### DISCUSSION THREAD

**Discussion by Geexp** [GarinZ/link-map#104] [General]
I really like Link Map and was hoping it would replace Tabs Outliner for me.

But the last version was released in April 2023, and there doesn't seem to be any activity on Github. 

It makes me very dubious in paying for the Pro Licence. 

Is this dead?

**Comment by Decioy** [comment]
Chrome webstore states that the last version is from March 20th, 2024, although the same v.1.1.6 from 2023...
